test/src/module/common/lockTest.c

```diff
@@ -24,22 +24,33 @@ testRun(void)
         int lockFdTest = -1;
 
         TEST_RESULT_INT(system(strZ(strNewFmt("touch %s", strZ(archiveLock)))), 0, "touch lock file");
-        TEST_ASSIGN(lockFdTest, lockAcquireFile(archiveLock, 0, true), "get lock");
+        TEST_ASSIGN(lockFdTest, lockAcquireFile(archiveLock, STRDEF("1-test"), 0, true), "get lock");
```

Comment:
General comment on the lockTest.c - since we store the PID and the EXECID (which is a random number plus the main PID which may not be the same as the current PID) it's unclear to me if we need a test to ensure the lock file contains what we **think** it should contain...